src/core/operations/YaraRules.mjs

```diff
@@ -0,0 +1,82 @@
+/**
+ * @author Matt C [matt@artemisbot.uk]
+ * @copyright Crown Copyright 2019
+ * @license Apache-2.0
+ */
+
+import Operation from "../Operation";
+import OperationError from "../errors/OperationError";
+import Yara from "libyara-wasm";
+
+/**
+ * Yara Rules operation
+ */
+class YaraRules extends Operation {
+
+    /**
+     * YaraRules constructor
+     */
+    constructor() {
+        super();
+
+        this.name = "Yara Rules";
+        this.module = "Yara";
+        this.description = "Yara support";
```

Comment:
Probably want to flesh this out a bit! Include a description of what Yara is and how this operation works.